website/docs/components/form/super-select/partials/code/how-to-use.md

```diff
@@ -126,9 +121,9 @@ To group similar sets of options, pass a nested data structure specifying the `g
 </Hds::Form::SuperSelect::Multiple::Field>
 ```
 
-#### With rich content in options
+### Rich content options
```

Comment:
Maybe "Options with rich content"?

To me, "Rich content options" sounds a bit like a special option type (ie richContentOption vs. plainContentOption or something) vs. just the content of the options.

Reply:
In the guidelines @majedelass refers to "rich content within option items". I think that could make sense for this header too as another option.

Reply:
One of the things I noticed was that most of the properties were related to `option`, so if you look at the headings holistically, now they are more similar in the wording- `{{adjective}} options` 

WDYT? open to changing that approach if that's what we want. 

(Screenshot to demonstrate: 

![the on this page submenu](https://github.com/hashicorp/design-system/assets/4587451/f9938613-a033-4379-8f0f-1e97ed9b87e0)